src/modules/cp/components/VisitorSpaceSettings/VisitorSpaceSettings.tsx

```diff
@@ -10,7 +10,7 @@ import { useHasAllPermission } from '@shared/hooks/has-permission';
 import { useLocale } from '@shared/hooks/use-locale/use-locale';
 import useTranslation from '@shared/hooks/use-translation/use-translation';
 import { toastService } from '@shared/services/toast-service';
-import { isBrowser } from '@shared/utils';
+import { NoServerSideRendering } from '@visitor-space/components/NoServerSideRendering/NoServerSideRendering';
```

Comment:
Missing barrel(s)?

Reply:
yeah im not a fan of barrel files
https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/